presto-memory/src/main/java/com/facebook/presto/plugin/memory/MemoryMetadata.java

```diff
@@ -204,6 +205,14 @@ public synchronized MemoryOutputTableHandle beginCreateTable(ConnectorSession se
         return new MemoryOutputTableHandle(table, ImmutableSet.copyOf(tableIds.values()));
     }
 
+    private void addSchemaIfNotExists(SchemaTableName tableName)
+    {
+        String schemaName = tableName.getSchemaName();
+        if (!schemas.contains(schemaName)) {
```

Comment:
IMO it should raise an exception that given schema does not exist. Instead user should explicitely create schema (see createSchema, dropSchema methods in ConnectorMetadata)